website/content/partials/oidc-jwt-claims.mdx

```diff
@@ -0,0 +1,73 @@
+### Bound claims
```

Comment:
In general, we use partials to allow content reuse across multiple docs. This partial is *really* long, which makes me think that it might not be the right solution in this case.

Two questions:
(1) Do we think the information here would be used anywhere other than the two pages created in this PR?
(2) If so, are there cases where it would be useful to include one of the sections (e.g., "Claims as metadata") rather than the entire text?

If the answer to (1) or (2) is "no", I recommend just making this a full doc page that we can link to, rather than including the content directly.

If the answer to both questions is "yes", I recommend breaking this into multiple partials (1 section per partial) so we can use the text modularly.

I recommend breaking it into 3 partials (one section per partial)..

Reply:
> (1) Do we think the information here would be used anywhere other than the two pages created in this PR?

No, I don't see this content being used outside of these two pages. This partial is equally relevant to both OIDC and JWT auth methods. Prior to splitting the two in this PR, it was included as a [single section](https://developer.hashicorp.com/vault/docs/auth/jwt#bound-claims).

Would you prefer I make it a separate page?